common/src/main/java/org/opensearch/ml/common/connector/AbstractConnector.java

```diff
@@ -65,11 +65,18 @@ public abstract class AbstractConnector implements Connector {
     protected User owner;
     @Setter
     protected AccessMode access;
+    @Setter
```

Comment:
Since you're only allowing setting the updated time remotely and setting created time internally you can probably omit this setter.

Reply:
As stated in a previous comment regarding the abstract connector we onboarded  `void setCreatedTime(Instant createdTime); void setLastUpdateTime(Instant lastUpdateTime);` so Abstract connector will need to implement these using lombok setters. In this way we can `set the time on indexing`